os/include/tinyara/ble/ble_manager.h

```diff
@@ -0,0 +1,79 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef _TIZENRT_BLE_DRV_MANAGER_H__
+#define _TIZENRT_BLE_DRV_MANAGER_H__
+
+#include <tinyara/config.h>
+#include <time.h>
```

Comment:
I think this is not needed. Please remove this.

Reply:
I fixed it